lib/index.js

```diff
@@ -204,13 +207,9 @@ module.exports.render = function(options) {
   var importer = options.importer;
 
   if (importer) {
-    options.importer = function(file, prev, key) {
+    options.importer = function(file, prev, bridge) {
       function done(data) {
-        console.log(data); // ugly hack
```

Comment:
Oops, I must have removed this thinking this was part of my debug logs. But now I'm intrigued, is there an issue ticket that explains why this hack was introduced?